editor/src/components/canvas/camvas-floating-toolbars.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
```

Comment:
This file is called `camvas-floating-toolbars.tsx` BTW...

Reply:
oh boy